docs/Using-Fleet/Adding-hosts.md

```diff
@@ -1,40 +1,61 @@
+  - [Deploy the TLS certificate that osquery will use to communicate with Fleet](#deploy-the-tls-certificate-that-osquery-will-use-to-communicate-with-fleet)
+  - [Launching osqueryd](#launching-osqueryd)
+  - [Using a flag file to manage flags](#using-a-flag-file-to-manage-flags)
+    - [Flag file on Windows](#flag-file-on-windows)
+  - [Migrating from plain osquery to osquery installer](#migrating-from-plain-osquery-to-osquery-installer)
+    - [Generate installer](#generate-installer)
+    - [Migrate](#migrate)
+- [Grant full disk access to osquery on macOS](#grant-full-disk-access-to-osquery-on-macos)
+  - [Creating the configuration profile](#creating-the-configuration-profile)
+    - [Obtaining identifiers](#obtaining-identifiers)
+    - [Creating the profile](#creating-the-profile)
+    - [Test the profile](#test-the-profile)
+
+## Introduction
+
+Fleet gathers information from an [osquery](https://github.com/osquery/osquery) agent installed on each of your hosts. The recomended way to install osquery is using [Orbit](./Orbit.md), Fleet's lightweight osquery runtime and autoupdater. 
+
+You can also install plain osquery on your hosts and connect to Fleet using osquery's `TLS API` plugins
+## Add hosts with Orbit
+
+To create an Orbit installer, you can use the `fleetctl package` command. To use the `fleetctl package` command, you must first install the `fleetctl` command-line tool. Instructions for installing `fleetctl` can be found [here](https://fleetdm.com/fleetctl-preview)
+
+`fleetctl package` can be used to create an Orbit installer which adds macOS hosts (**.pkg**), Windows hosts (**.msi**), or Linux hosts (**.deb** or **.rpm**) to Fleet.
+
+The following command creates an Orbit installer, `.pkg` file, which adds macOS hosts to Fleet. Locate this osquery installer in the folder where the `fleetctl package` command is run.
 
 ```sh
 fleetctl package --type pkg --fleet-url=[YOUR FLEET URL] --enroll-secret=[YOUR ENROLLMENT SECRET]
 ```
   >**Note:** The only configuration option required to create an installer is `--type`, but to communicate with a Fleet instance, you'll need to specify a `--fleet-url` and `--enroll-secret`
 
-When you install the generated osquery installer on a host, this host will automatically enroll in the specified Fleet instance.
+When you install the generated Orbit installer on a host, this host will automatically enroll in the specified Fleet instance.
 
 ### Signing installers
 
-  >**Note:** Currently, the fleetctl package command does not provide support for signing Windows osquery installers. Windows installers can be signed after building.
+  >**Note:** Currently, `fleetclt package` does not provide support for signing Windows Orbit installers. Windows installers can be signed after building.
+
+`fleetctl package` provides support for signing and notarizing macOS osquery installers via the
+`--sign-identity` and `--notarize` flags.- [Orbit installer](#orbit-installer)
```

Comment:
@ksatter, is "Orbit  installer" supposed to be at the of this sentence? It doesn't seem to be connected.